package.json

```diff
@@ -13,10 +13,10 @@
   "dependencies": {
     "react": "18.0.0",
     "react-dom": "18.0.0",
-    "react-scripts": "4.0.3"
+    "react-scripts": "^5.0.1"
   },
   "devDependencies": {
```

Comment:
**nitpick** If we run `yarn build`, we'll receive a warning about an outdated `browserslist` version.

```shell
Browserslist: caniuse-lite is outdated. Please run:
  npx update-browserslist-db@latest
  Why you should do it regularly: https://github.com/browserslist/update-db#readme
```

**suggestion** it would be great to update it. We can do this with the following command:

```shell
yarn add -D browserslist@^4.21.0
```